packages/kit/src/runtime/client/singletons.js

```diff
@@ -1,4 +1,4 @@
-/** @type {import('./router').Router?} */
+/** @type {import('./router').Router} */
```

Comment:
I think we can add a jsdoc comment here mentioning this is nullable, but is not typed as so to prevent extra type checks.

Reply:
I've updated it to address this